Mathlib/Topology/CompactOpen.lean

```diff
@@ -255,6 +255,25 @@ instance [T3Space Y] : T3Space C(X, Y) := inferInstance
 
 end Ev
 
+section DiscreteTopology
+variable [DiscreteTopology X]
+
+/-- The continuous functions from `X` to `Y` are the same as the plain functions when `X` is
+discrete. -/
+@[simps!]
+def homeoFnOfDiscrete : C(X, Y) ≃ₜ (X → Y) where
```

Comment:
Does any other homeomorph definition use `homeo`, `not `homeomorph` in the name?

Reply:
Yes, eg `TopCat.homeoOfIso`. I'm sure there are more